frontend/src/features/review/components/coverLetter/CoverLetterContent.tsx

```
@@ -0,0 +1,68 @@
+  });
+
+  return (
+    <div
+      ref={containerRef}
+      onMouseUp={handleMouseUp}
+      className='relative min-h-0 w-full flex-1 overflow-y-auto pr-[2rem] pb-68 pl-[3rem]'
+      style={{
+        whiteSpace: 'pre-wrap',
+        overflowY: selection ? 'hidden' : 'auto',
+      }}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)